### ISSUE TITLE
TensorRT EP same inference Time of INT 8 and FP 16 

### ISSUE THREAD

**Issue by eric-tc** [microsoft/onnxruntime#14315]
### Describe the issue

Hi,

I have made a simple test on Resnet152 model. I have quantize the model and get  onnx Q/DQ. Onnx models are available here.

https://drive.google.com/file/d/1uRNyexXARUJSyIyu7Zww_S-FgfOO5DbA/view?usp=share_link resnet152 quantize 
https://drive.google.com/file/d/1P7uqkdJUCNUN3Q2vLFv0hGS3pGHQFl6S/view?usp=share_link normal

When i try to run inference using TensoRT Execution Provider I got more or less the same inference time for 
INT8 and FP16. I am using an RTX 2070 which is supposed to have Tensor Core with INT8 optimization.

I would like to know if this is an expected behavior or I am missing something.

### To reproduce

import onnxruntime as ort
import torch
import time

#export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/usr/local/lib/python3.8/dist-packages/tensorrt/

def to_numpy(tensor):
    return tensor.detach().cpu().numpy() if tensor.requires_grad else tensor.cpu().numpy()
    
providers = [
    ('TensorrtExecutionProvider', {
        'device_id': 0,
        'trt_max_workspace_size': 2147483648,
        #'trt_fp16_enable': True,  #will change this if I have a quantized model or not
         'trt_int8_enable': True
    }),
    ('CUDAExecutionProvider', {
        'device_id': 0,
        'arena_extend_strategy': 'kNextPowerOfTwo',
        'gpu_mem_limit': 2 * 1024 * 1024 * 1024,
        'cudnn_conv_algo_search': 'EXHAUSTIVE',
        'do_copy_in_default_stream': True,
    })
]

sess_opt = ort.SessionOptions()

ort_session = ort.InferenceSession("quant_resnet152.onnx", providers=providers)

tensor= torch.randn(1,3,224,224)

ort_inputs = {ort_session.get_inputs(
        )[0].name: to_numpy(tensor)}

outputs= ort_session.run(None,ort_inputs)

start= time.time()
for i in range(0,100,1):
    outputs= ort_session.run(None,ort_inputs)

stop= time.time()

print("total time {}".format(stop-start))
print("single inference {}".format((stop-start)/100))

// RESULTS

MODEL INT 8  total time 0.27687644958496094
single inference 0.0027687644958496096

MODEL FP16 total time 0.22818613052368164
single inference 0.0022818613052368163

### Urgency

None

### Platform

Windows

### OS Version

ubuntu 18.04

### ONNX Runtime Installation

Released Package

### ONNX Runtime Version or Commit ID

1.13.1

### ONNX Runtime API

Python

### Architecture

X64

### Execution Provider

TensorRT

### Execution Provider Library Version

Tensorrt 8.5 Cuda 11.7 cudnn 8.5

**Comment by stevenlix** [comment]
First, please enable 'trt_fp16_enable' along with 'trt_int8_enable'. For some kernels FP16 may be faster than INT8.
If perf is still the same, please try original FP32 model plus calibration table rather than QDQ approach so that TRT can have more freedom to decide how to do INT8 quantization for CNN models. What you need to do is to feed in original FP32 model and provide calibration table flatbuffer file location by trt_int8_calibration_table_name in TRT provider options
